src/api/uploads/__tests__/BaseUpload.test.js

```diff
@@ -173,6 +173,26 @@ describe('api/uploads/BaseUpload', () => {
             expect(upload.uploadsReachability.isReachable).not.toHaveBeenCalled();
         });
 
+        test('should call preflightSuccessHandler, without doing a reachability test if DEFAULT_HOSTNAME_UPLOAD_APP is provided as uploadUrl', () => {
```

Comment:
i just noticed this, its minor so ok to leave as is, but we can avoid test duplication by using test.each.